src/Microsoft.Maintainability.Analyzers/UnitTests/AvoidUnusedPrivateFieldsTests.Fixer.cs

```diff
@@ -12,22 +14,92 @@ public class AvoidUnusedPrivateFieldsFixerTests : CodeFixTestBase
     {
         protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
         {
-            return new BasicAvoidUnusedPrivateFieldsAnalyzer();
+            return new AvoidUnusedPrivateFieldsAnalyzer();
         }
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpAvoidUnusedPrivateFieldsAnalyzer();
+            return new AvoidUnusedPrivateFieldsAnalyzer();
         }
 
         protected override CodeFixProvider GetBasicCodeFixProvider()
         {
-            return new BasicAvoidUnusedPrivateFieldsFixer();
+            return new AvoidUnusedPrivateFieldsFixer();
         }
 
         protected override CodeFixProvider GetCSharpCodeFixProvider()
         {
-            return new CSharpAvoidUnusedPrivateFieldsFixer();
+            return new AvoidUnusedPrivateFieldsFixer();
+        }
+
+        [Fact]
+        public void CA1823CSharp()
+        {
+            VerifyCSharpFix(
+                @"  
+class C  
+{  
```

Comment:
I think we should have tests for multiple fields declared on the same line, for example:

```
private int used1, unused1, unused2, used2, unused3;
```

should become

```
private int used1, used2;
```